### DISCUSSION TITLE
Nextra i18n's fate?

### DISCUSSION THREAD

**Discussion by syahzuan** [shuding/nextra#1005] [General]
I just found Nextra today, yet to set up a site.
Currently a Next.js user, and Next 13 scrapping the existing i18n has caused a bit of an issue for us.
If Nextra is a Next.js framework, can I assume its existing i18n feature is going away as well?
Asking because i18n is an important feature for our site, and it'd be good to know how Nextra handles it before we pick it up.
Cheers.

**Comment by shuding** [comment]
Hi! First of all Next.js 13 isn't removing the i18n feature (I'm also one of Next.js maintainers). 

For pages/ folder everything remains the same, you can still use the `i18n` option in next.config.js and everything should work fine. For app/ we are going to add i18n support with a new approach, that works better in the new modal.

And since Nextra will stay with the pages/ folder, i18n will not be affected at all.
